samples/hosting/generic-multi-hosting/sample.md

```diff
@@ -0,0 +1,22 @@
+---
+title: Generic Host multiple endpoint hosting
+summary: Hosting multiple endpoints in a generic host process.
+reviewed: 2020-08-10
+component: Core
+related:
```

Comment:
Are redirects automagically created or do we need to add a redirect that points back to the previous URI we used?

Reply:
Done